frontend/src/components/Footer/Footer.js

```diff
@@ -3,32 +3,35 @@ import Instagram from "../Client/Social/Instagram"
 
 const Footer = () => {
   return (
-    <div className="flex flex-col mt-auto bg-indigo-400 text-white">
-      <div className="mx-auto w-full max-w-6xl">
-        <div className="flex flex-col md:flex-row md:justify-between md:mr-16">
-          <div className="flex justify-around pt-4 md:w-96 md:items-center">
-            <div className="flex flex-col">
-              <h5 className="text-xl mb-2">ABOUT</h5>
-              <LinkWrapper to="#" title="Reviews" />
-              <LinkWrapper to="#" title="Our Story" />
-              <LinkWrapper to="#" title="Careers" />
+    <>
+      <div className='bg-transparent h-8'></div>
```

Comment:
whats this do?

Reply:
I explained it in my PR description. It's a transparent div with height-8 placed above 'footer' to serve as the margin between any content and the footer. Directly adding top margin to the footer caused it to no longer stay fixed to the bottom of any page. I couldn't find another way to do it.